Add ability to get comments from a post (#6)
stanfordio/gogettr: README.md

  --help  Show this message and exit.
```

> ### `comments`

Reviewer: I've been trying to keep the order here matching the output of `gogettr --help`, so this should be the second entry?

```text

> Usage: gogettr comments [OPTIONS] POST ID
> 
>   Pull comments from a post
> 
> Options:
>   --max INTEGER  the maximum number of users to pull
>   --help  Show this message and exit.

Reviewer: ```suggestion
Usage: gogettr comments [OPTIONS] POST_ID

  Pull comments on a specific post.

Options:
  --max INTEGER  the maximum number of comments to pull
  --help         Show this message and exit.
```

I usually just copy and paste the output of `gogettr comments --help` here.

Support for the following capabilities is planned:

> - [x] Pull all comments for a post

Reviewer: We could honestly just remove this line, now that it's implemented! :)
